### PR TITLE
core: add Text panel

### PR DESCRIPTION
**Pull request by filippog**

### CODE REVIEW

File: grafanalib/core.py [hunk 1]
```diff
@@ -893,6 +894,35 @@ def to_json_data(self):
 
 
 @attr.s
+class Text(object):
```
Comment (jml) [inline grafanalib/core.py]: Please add a docstring, e.g.

```python
"""A Text panel."""
```

File: grafanalib/core.py [hunk 2]
```diff
@@ -893,6 +894,35 @@ def to_json_data(self):
...
+    content = attr.ib()
+    editable = attr.ib(default=True, validator=instance_of(bool))
+    error = attr.ib(default=False, validator=instance_of(bool))
+    height = attr.ib(default=None)
+    id = attr.ib(default=None)
+    links = attr.ib(default=attr.Factory(list))
+    mode = attr.ib(default="markdown")
```
Comment (jml) [inline grafanalib/core.py]: What's the range of values for this? We should probably have a set of constants, as we do for types, evaluators, etc.

Comment (filippog) [inline grafanalib/core.py]: indeed, afaict one of html/markdown/text, I've turned that into constants

**Comment by filippog** [comment]
@jml added changelog entry too, PTAL!

**Pull request merged**